server/js/sockets.js

```diff
@@ -174,6 +174,13 @@ sockets.init = function socketInit(server) {
       );
     });
 
+    socket.on('send-message', message => {
+      mqttClient.publish(
+        '/v3/camera/primary/message',
+        `message=${message}`,
```

Comment:
Why do we put the title `message=` here rather than in the base layer of the overlay? Even if we do have `message=` as part of the message, rather than in the base layer, I think it should be prepended in the overlay code, not here.

Reply:
This is meant to json encode the payload. See the V3 MQTT spec.

Reply:
Good point. Though that's not valid json though is it?

Reply:
Related to this, can we remove the `message=` ? It's unnecessary to be shown on the overlay.
![93695750_245548110042305_5773327513530925056_n](https://user-images.githubusercontent.com/18709969/79688918-ec553800-8294-11ea-86bb-ed9947e65ea5.png)

Reply:
@khanguslee If you checkout monash-human-power/raspicam#28, "Message:" only appears once.

Reply:
As discussed in #28 , let's keep it like that. I believe @maanika's reason is pretty good.

Reply:
To summarise what's been said here
- `message=` should not be put here, instead the title `Message: ` should be added in monash-human-power/raspicam#28.
- This packet should be formated as JSON (see the page "V3 MQTT Topics" on Notion)